config/service.yml

```diff
@@ -86,4 +86,20 @@ services:
     arguments:
       - '@mollie'
 
-  Mollie\Service\CartDuplicationService:
\ No newline at end of file
+  Mollie\Service\CartDuplicationService:
+
+  Mollie\Service\TransactionService:
+    arguments:
+      - '@mollie'
+      - '@Mollie\Repository\PaymentMethodRepository'
+      - '@Mollie\Service\OrderStatusService'
+
+  Mollie\Service\MolliePaymentMailService:
+    arguments:
+      - '@mollie'
+      - '@Mollie\Repository\PaymentMethodRepository'
+      - '@Mollie\Service\MailService'
+
+  Mollie\Service\MailService:
+    arguments:
+      - '@mollie'
```

Comment:
no new line at the end of file. @margud  can you copy this file https://bitbucket.org/gourmetli/admin-frontend/pull-requests/69/in-1001-feature-moves-e2e-tests-part1 in mollie's module and in your IDE, sample applies for @lukvai  add editorconfig plugin ?